src/modules/cpu/kernel/glitch.hpp

```diff
@@ -0,0 +1,673 @@
+        {
+            Rpp8u *dstPtrRow;
+            dstPtrRow = dstPtrChannel;
+            Rpp32u alignedLength = ((int)((roi.xywhROI.roiWidth * 0.75)) / 8) * 8;   // Considering 75% of the ROI width to avoid loading out-of-bounds locations.
+            for (int dstLocRow = 0; dstLocRow < roi.xywhROI.roiHeight; dstLocRow++)
+            {
+                Rpp8u* dstRowPtrTempR = dstPtrRow;
+                Rpp8u* dstRowPtrTempG = dstPtrRow + dstDescPtr->strides.cStride;
+                Rpp8u* dstRowPtrTempB = dstPtrRow + 2 * dstDescPtr->strides.cStride;
+
+                for (int vectorLoopCount = 0; vectorLoopCount < alignedLength; vectorLoopCount += 8)
+                {
+                    __m256 p[3];
+                    compute_src_loc(dstLocRow, vectorLoopCount, glitchSrcLocArray, srcDescPtr, rgbOffsets, roi, batchCount, 3);
+                    rpp_simd_load(rpp_glitch_load24_u8pkd3_to_f32pln3_avx, srcPtrChannel, p, glitchSrcLocArray);
+                    rpp_simd_store(rpp_store24_f32pln3_to_u8pln3_avx, dstRowPtrTempR, dstRowPtrTempG, dstRowPtrTempB, p);    // simd stores
+
+                    dstRowPtrTempR += 8;
+                    dstRowPtrTempG += 8;
+                    dstRowPtrTempB += 8;
+                }
+
+                for (int i = alignedLength; i < roi.xywhROI.roiWidth; i++)
+                {
+                    compute_src_loc(dstLocRow, i, glitchSrcLocArray, srcDescPtr, rgbOffsets, roi, batchCount, 3);
+                    *dstRowPtrTempR++ = *(srcPtrChannel + glitchSrcLocArray[0] + 0);
+                    *dstRowPtrTempG++ = *(srcPtrChannel + glitchSrcLocArray[1] + 1);
+                    *dstRowPtrTempB++ = *(srcPtrChannel + glitchSrcLocArray[2] + 2);
+                }
+
+                dstPtrRow += dstDescPtr->strides.hStride;
+            }
+        }
+        else if ((srcDescPtr->c == 3) && (srcDescPtr->layout == RpptLayout::NCHW) && (dstDescPtr->layout == RpptLayout::NHWC))
+        {
+            Rpp8u *dstPtrRow;
+            dstPtrRow = dstPtrChannel;
+            Rpp32u vectorIncrement = 16;
+            Rpp32u alignedLength = ((int)(roi.xywhROI.roiWidth * 0.75)) & ~15; // Considering 75% of the ROI width to avoid loading out-of-bounds locations.
+
+            for (int dstLocRow = 0; dstLocRow < roi.xywhROI.roiHeight; dstLocRow++)
+            {
+                Rpp8u* dstPtrTemp = dstPtrRow;
+
+                for (int vectorLoopCount = 0; vectorLoopCount < alignedLength; vectorLoopCount += 16)
+                {
+                    __m256 p[6];
+                    compute_src_loc(dstLocRow, vectorLoopCount, glitchSrcLocArray, srcDescPtr, rgbOffsets, roi, batchCount, 1);
+                    rpp_simd_load(rpp_load48_u8pln3_to_f32pln3_avx, srcPtrChannel+glitchSrcLocArray[0], srcPtrChannel + srcDescPtr->strides.cStride + glitchSrcLocArray[1], srcPtrChannel + 2 * srcDescPtr->strides.cStride + glitchSrcLocArray[2], p);
```

Comment:
if below are constant, would suggest precompute and store in a buffer prior and use it here instead of doing this operation everytime

```
srcDescPtr->strides.cStride + glitchSrcLocArray[1]
2 * srcDescPtr->strides.cStride + glitchSrcLocArray[2]
```

Reply:
Done